app/assets/stylesheets/application.scss

```diff
@@ -232,6 +240,10 @@
   box-shadow: 0 0 0 3pt $yellow;
 }
 
+.new-search {
+  display: none;
```

Comment:
SO if you don't have JS you will never see these options... :(

Reply:
sorted :)